python/lsst/sconsUtils/state.py

```diff
@@ -98,8 +98,8 @@ def _initVariables():
         SCons.Script.BoolVariable('force', 'Set to force possibly dangerous behaviours', False),
         ('optfile', 'Specify a file to read default options from', None),
         ('prefix', 'Specify the install destination', None),
-        SCons.Script.EnumVariable('opt', 'Set the optimisation level', 0,
-                                  allowed_values=('0', '1', '2', '3')),
+        SCons.Script.EnumVariable('opt', 'Set the optimisation level', 3,
+                                  allowed_values=('g', '0', '1', '2', '3')),
```

Comment:
I doubt we'd ever really want fast as it always seems to have so many caveats about it and does not seem to be safe for algorithmic work. I doubt anyone would really need `s` either so probably can just leave it. 4 seems to be the same as 3.